lib/backend/k8s/k8s.go

```diff
@@ -728,7 +729,7 @@ func (c *KubeClient) getGlobalConfig(k model.GlobalConfigKey) (*model.KVPair, er
 }
 
 // listGlobalConfig lists all global configs.
-func (c *KubeClient) listGlobalConfig(l model.GlobalConfigListOptions) ([]*model.KVPair, error) {
+func (c *KubeClient) listGlobalConfig(l model.GlobalConfigListOptions) ([]*model.KVPair, error, string) {
```

Comment:
I'm reverting this anyway - don't want to introduce any extra risk into this change.